scripts/get_openstack_plaintext_docs.sh

```diff
@@ -11,7 +11,8 @@ fi
 OS_VERSION=${OS_VERSION:-2024.2}
 
 # List of OpenStack Projects
-_OS_PROJECTS="nova neutron cinder keystone glance swift"
+_OS_PROJECTS="nova horizon keystone neutron cinder manila glance swift ceilometer \
```

Comment:
Watcher is missing (soon to be introduced).

How are we planning to track here components that are to be added/removed from docs?

Reply:
Thanks for the recommendation. I've added the watcher into the default set of projects.

> How are we planning to track here components that are to be added/removed from docs?

For now I will update the ticket. The main question is whether we want to include ALL services or only those supported in RHOSO env. I'd like to also add the openstackclient documentation. But I'll do that in a separate PR. I started with services above because I had concerns about the GitHub Actions 6 hour limit. It seem like we have still a room for now (the job took ~ 3 hours).